translations/binaries/slps_strings_group_08.json

```
@@ -26,7 +26,7 @@
             "string": "狙う敵を毎回変え、フィールドを[LINE]\n駆け回ります。[END]",
             "original_bytes_length": 46,
             "available_bytes_size": 47,
-            "translation": "노릴 적을 매번 바꾸며, 필드를[LINE]\n뛰어다닙니다.[END]"
+            "translation": "타킷을 매번 바꾸며, 필드를[LINE]\n뛰어다닙니다.[END]"
```

타깃(Target)의 오타일까요?

현대 한국어도 외래어를 많이 쓰는 편이긴 하지만, 한국어 표현이 가능하다면 그 쪽을 우선하면 좋은 것 같습니다.

```suggestion
            "translation": "대상을 매번 바꾸며, 필드를[LINE]\n뛰어다닙니다.[END]"
```